### PR TITLE
endpoint-post-categories/ot289-98

### PR DESCRIPTION
**Pull request by danielDeVita**
jira link:
https://alkemy-labs.atlassian.net/browse/OT289-98

summary:
and endpoint to POST new categories was implemented with validations from express validator

evidence:
https://user-images.githubusercontent.com/99274893/194964561-d5e755d9-d627-4c98-aba6-bf2bfc81e475.mov

### CODE REVIEW

File: controllers/categoriesController.js [hunk 1]
```diff
@@ -3,39 +3,56 @@ const { validationResult } = require("express-validator");
...
+    postCategory: (req, res) => {
+        const errors = validationResult(req)
+        if (!errors.isEmpty()) {
+            return res.json({ errors: errors.mapped() });
+        } else {
+            db.Categories.create({
+                name: req.body.name,
+            })
+                .then((categorie) => {
+                    return res.status(200).json({ message: "Categorie created succesfully", categorie })
```
Comment (GauntStrelok) [inline controllers/categoriesController.js]: Categoria en ingles es category y no "categorie"

**Comment by GauntStrelok** [comment]
Hacer pull de main para arreglar conflictos

**Comment by danielDeVita** [comment]
solucionado aca tambien, mis disculpas!

**Comment by danielDeVita** [comment]
https://user-images.githubusercontent.com/99274893/196309721-db8bf53e-7dc4-4f39-8294-9b2851a98169.mp4

new evidence, everything working!

**Pull request merged**